int32    index
int32    addr
int32    count
string   valTab
string   valType
---
int32 res